compiler/crates/relay-compiler/tests/compile_relay_artifacts/fixtures/connection-name-matches-fragment.invalid.expected

```diff
@@ -0,0 +1,630 @@
+                    "name": "pageInfo",
+                    "plural": false,
+                    "selections": [
+                      {
+                        "alias": null,
+                        "args": null,
+                        "kind": "ScalarField",
+                        "name": "endCursor",
+                        "storageKey": null
+                      },
+                      {
+                        "alias": null,
+                        "args": null,
+                        "kind": "ScalarField",
+                        "name": "hasNextPage",
+                        "storageKey": null
+                      }
+                    ],
+                    "storageKey": null
+                  }
+                ],
+                "storageKey": "comments(first:2)"
+              },
+              {
+                "alias": null,
+                "args": [
+                  {
+                    "kind": "Literal",
+                    "name": "first",
+                    "value": 2
+                  }
+                ],
+                "filters": null,
+                "handle": "connection",
+                "key": "NodeQuery_comments",
+                "kind": "LinkedHandle",
+                "name": "comments"
+              }
+            ],
+            "type": "Story",
+            "abstractKey": null
+          }
+        ],
+        "storageKey": "node(id:\"4\")"
+      }
+    ]
+  },
+  "params": {
+    "cacheID": "6a1f1d6e57db7197e17decd6c2039740",
+    "id": null,
+    "metadata": {},
+    "name": "connectionNameMatchesFragmentQuery",
+    "operationKind": "query",
+    "text": null
+  }
+}
+
+QUERY:
+
+query connectionNameMatchesFragmentQuery {
+  ...connectionNameMatchesFragmentFragment
+}
+
+fragment connectionNameMatchesFragmentFragment on Query {
+  node(id: "4") {
+    __typename
+    id
+    ... on Story {
+      comments(first: 2) {
+        edges {
+          node {
+            id
+            __typename
+          }
+          cursor
+        }
+        pageInfo {
+          endCursor
+          hasNextPage
+        }
+      }
+    }
+  }
+}
+
+
+{
+  "argumentDefinitions": [
+    {
+      "defaultValue": null,
+      "kind": "LocalArgument",
+      "name": "after"
+    },
+    {
+      "defaultValue": 2,
+      "kind": "LocalArgument",
+      "name": "first"
+    }
+  ],
+  "kind": "Fragment",
+  "metadata": {
+    "connection": [
+      {
+        "count": "first",
+        "cursor": "after",
+        "direction": "forward",
+        "path": [
+          "node",
+          "comments"
+        ]
+      }
+    ],
+    "refetch": {
+      "connection": {
+        "forward": {
+          "count": "first",
+          "cursor": "after"
+        },
+        "backward": null,
+        "path": [
+          "node",
+          "comments"
+        ]
+      },
+      "fragmentPathInResult": [],
+      "operation": require('connectionNameMatchesFragmentFragment.graphql')
```

Comment:
This is the problem. We meant to import the generated operation's artifact, but we accidentally got the fragment's.